charts/k8s-monitoring/tests/platform/eks-with-windows/.rendered/output.yaml

```diff
@@ -979,6 +980,159 @@ data:
       namespace = "default"
     }
     
+    // Feature: Node Logs
```

Comment:
Presume you'd need to maintain this every time the node logs module is updated?

Reply:
Alloy modules are synced and kept locally. So whenever they're updated, it'll update the examples and rendered test deployments. No manual steps required.